docs/operator/README.md

```diff
@@ -547,7 +493,7 @@ In addition to the Polymesh metrics you should also monitor basic node metrics a
 | Free disk space           | 30 GB+ or > 20% volume capacity | There should always be some free disk space for the Polymesh node to consume. |
 | Free RAM                  | 1 GB+ | Spikes in RAM usage are acceptable but on average, there should be at least 1 GB of free RAM available on the system for the node to consume.|
 | CPU usage                 | 5-50% (overall) | This is the overall CPU usage and not per core usage. Occasional spikes above 50% are acceptable but more cores should be added if the CPU usage continuously stays above 50%. |
-| Network connectivity      | <1% packet loss | This mainly applies to sentry nodes. They should be online and reachable at all times. If they are being DDoS’d and can not respond to queries, new sentry nodes should be deployed. |
+| Network connectivity      | <1% packet loss | Nodes should be online and reachable at all times. If they are being DDoS’d and can not respond to queries, new nodes should be deployed, or the operators connectivity limited to trusted nodes. |
```

Comment:
```suggestion
| Network connectivity      | <1% packet loss | Nodes should be online and reachable at all times. If nodes are being DDoSed and cannot respond to queries then either new nodes should be deployed or connectivity should be limited to trusted nodes using firewall rules and the `--reserved-only` and `--reserved-nodes` parameters. |
```

Reply:
I think we should drop the "<1% packet loss" since we do not have a direct way to collect that data. Instead, we can say something like ">0.1 mbps bandwidth". Basically, we just want to ensure that the nodes are able to talk to each other.

ps We need to revamp the full monitoring/alerting section anyway.